src/views/v2/commerce/gateway/stripe/card_element.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Buttons for Stripe
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/gateway/stripe/buttons.php
```

Comment:
```suggestion
 * [your-theme]/tribe/tickets/v2/commerce/gateway/stripe/card-element.php
```
We use `-` as separator for template files.